### PR TITLE
Corgi Gender Equality

### PR DESCRIPTION
**Pull request by LittleBigKid2000**
Makes corgis (Other than Ian, Lisa, and E-N) have a randomized gender, instead of always being male.
I'm sure nothing bad like accidental corgi puppy outbreaks could come from this.
~~Also fixes a grammar thing. The message you get when shaving a corgi will now properly use 'the' when the corgi doesn't have a name and is just "corgi".~~Nevermind

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 1]
```diff
@@ -84,7 +86,7 @@
 		if(shaved)
 			to_chat(user, "<span class='warning'>You can't shave this corgi, it's already been shaved!</span>")
 			return
-		user.visible_message("[user] starts to shave [src] using \the [O].", "<span class='notice'>You start to shave [src] using \the [O]...</span>")
+		user.visible_message("[user] starts to shave [src] using \the [O].", "<span class='notice'>You start to shave \the [src] using \the [O]...</span>")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm]: Noooo. @TheDZD is going to kill you for this.

**Comment by taukausanake** [comment]
inb4 we have a corgi pest outbreak. That should be a thing actually

**Comment by Plotron** [comment]
Dwarf Fortress-esque corgisplosion!

**Comment by GeneralGoose** [comment]
on tg, corgis don't have sex if you watch them and there's also pupper limit for each mother doggo

**Comment by taukausanake** [comment]
We should find that limit and remove it. I want Dwarf Fortress pet explosions

**Comment by Earthdivine** [comment]
I too, enjoy pet explosions as long as it's not Tribble related.

**Comment by Spacemanspark** [comment]
B-b-b-but muh tribbles

**Comment by KasparoVy** [comment]
The Trouble with Tribbles...

:scream:

**Pull request merged**